### PR TITLE
(PC-2438) : Add script to clean thumbCount

### PR DESCRIPTION
**Pull request by DanielBaudry**

### CODE REVIEW

File: tests/scripts/product_thumb/reset_thumb_count_test.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from scripts.product_thumb.reset_thumb_count import reset_thumb_count_before_processing_files
+from tests.conftest import clean_database
+from tests.test_utils import create_product_with_thing_type
+
+
+@clean_database
+def test_reset_thumb_count_before_processing_files(app):
+    # Given
+    for index in range(4):
+        PcObject.save(create_product_with_thing_type(thumb_count=randrange(5)))
```
Comment (NerOcrO) [inline tests/scripts/product_thumb/reset_thumb_count_test.py]:
```suggestion
        product = create_product_with_thing_type(thumb_count=randrange(5))
        PcObject.save(product)
```

File: scripts/product_thumb/reset_thumb_count.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from models import Product, PcObject
+
+
+def reset_thumb_count_before_processing_files():
```
Comment (NerOcrO) [inline scripts/product_thumb/reset_thumb_count.py]:
```suggestion
def reset_thumb_count():
```

File: scripts/product_thumb/reset_thumb_count.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    there_are_products_to_check = True
```
Comment (NerOcrO) [inline scripts/product_thumb/reset_thumb_count.py]:
```suggestion
    are_products_to_check = True
```

File: scripts/product_thumb/reset_thumb_count.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    while there_are_products_to_check:
+        products = Product.query \
+            .filter(Product.thumbCount > 0) \
+            .paginate(page=page_index, per_page=page_size, error_out=False).items
+        for product in products:
+            product.thumbCount = 0
+            product.firstThumbDominantColor = None
+        PcObject.save(*products)
+        page_index += 1
+        print(f"{page_index}")
```
Comment (NerOcrO) [inline scripts/product_thumb/reset_thumb_count.py]:
```suggestion
        print(f"page {page_index}")
```

File: tests/scripts/product_thumb/reset_thumb_count_test.py (hunk 1 above)
Comment (DanielBaudry) [reply tests/scripts/product_thumb/reset_thumb_count_test.py]: Ok

File: scripts/product_thumb/reset_thumb_count.py (hunk 2 above)
Comment (DanielBaudry) [reply scripts/product_thumb/reset_thumb_count.py]: Ok

File: scripts/product_thumb/reset_thumb_count.py (hunk 3 above)
Comment (DanielBaudry) [reply scripts/product_thumb/reset_thumb_count.py]: Ok

File: scripts/product_thumb/reset_thumb_count.py (hunk 4 above)
Comment (DanielBaudry) [reply scripts/product_thumb/reset_thumb_count.py]: Yes

**Pull request merged**